src/EventStore.Core/ClusterVNodeStartup.cs

```diff
@@ -140,7 +140,10 @@ IServiceProvider IStartup.ConfigureServices(IServiceCollection services) => Conf
 						.AddSingleton(new Operations(_mainQueue, _authorizationProvider))
 						.AddSingleton(new Gossip(_mainQueue, _authorizationProvider))
 						.AddSingleton(new Elections(_mainQueue, _authorizationProvider))
-						.AddGrpc().Services,
+						.AddGrpc()
+						.AddServiceOptions<Streams>(options =>
+							options.MaxReceiveMessageSize = TFConsts.MaxLogRecordSize - TFConsts.LogRecordOverheadSize)
```

Comment:
This might lead to some confusion when reading the code as we are using max log record size as the maximum payload size. Even if it's just for readability, could we not do something along the lines of introducing a constant
```
public const int MaxPayloadSize = MaxLogRecordSize - LogRecordOverheadSize;
```
and using that here?

Reply:
Perfect, I just couldn't figure out a good name for it :smile: